starters/tailwind/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "devDependencies": {
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.5.3",
+    "@storybook/addon-links": "^7.5.3",
+    "@storybook/addon-onboarding": "^1.0.8",
+    "@storybook/blocks": "^7.5.3",
+    "@storybook/react": "^7.5.3",
+    "@storybook/react-vite": "^7.5.3",
```

Comment:
Are we missing a dependency required by this package? I did an `yarn install` and `yarn storybook` and got the following error:

```=> Failed to build the preview
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'vite' imported from ./node_modules/@storybook/builder-vite/dist/index.js
```

I checked the node_modules and I have `@vitejs/`.